src/dxtbx/util/rotate_and_average.py

```diff
@@ -0,0 +1,32 @@
+import math
+import numpy as np
+
+
+def rotate_and_average(data, angle, deg=False, mask=None):
+    """
+    Angle is the rotation angle in the coordinate system of x along fast, y along slow, such that angle increases as it rotates around z, where z completes a right-handed system.
+    """
+    if deg:
+        angle *= math.pi / 180
+    angle = -angle
+
```

Comment:
@dwmoreau - chatting with @phyy-nx  right now and looking over this PR and thought this module could probably benefit from some tests 🙂